language-server/modules/langserver-core/src/test/resources/codeaction/change-var-type/source/change_var_type_in_object_fields.bal

```diff
@@ -0,0 +1,9 @@
+class TestClass {
+        
+    string f1 = 12.2;
+    
+    int a = "string";
+    
+    string arr = ["a", "b"];
```

Comment:
Shall we update this test case to have a qualifier like `private` (`private string arr = xxx`)?